src/systemTest/java/com/flightstats/hub/callback/CallbackCache.java

```diff
@@ -0,0 +1,37 @@
+package com.flightstats.hub.callback;
+
+import com.flightstats.hub.model.WebhookCallbackRequest;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Slf4j
+public class CallbackCache {
+
+    private final Map<String, List<String>> cacheRequestObject = new ConcurrentHashMap<>();
+
+    public void put(WebhookCallbackRequest webhookCallbackRequest) {
+        String webhookName = webhookCallbackRequest.getName();
+
+        synchronized (this) {
+            if (contains(webhookName)) {
+                get(webhookName).addAll(webhookCallbackRequest.getUris());
+            } else {
+                cacheRequestObject.put(
```

Comment:
I think we will have a need in future for this cache to handle multiple sets of values under the same webhook name, I adding some kind of further unique id to the key side of the pair would be necessary in the future if not now. Not suggesting it is in the scope of this PR necessarily, but worth a mention.